### PR TITLE
Adding Dice Game- project

### PR DESCRIPTION
**Pull request by Ritu-2001**
This a mini project using html,css and Javascript

### CODE REVIEW

**Comment by Ritu-2001** [comment]
add My project please check it and merge it

**Comment by Ritu-2001** [comment]
project is added  check it and please merge it

File: Dicee Challenge - Starting Files/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+var randomNumber1=Math.floor(Math.random()*6+1);
```
Comment (aaryan2134) [inline Dicee Challenge - Starting Files/index.js:1]: Please add comments to explain what you are doing here

File: Dicee Challenge - Starting Files/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+This is a begginer Friendly Mini project using Html,CSS and Javascript. In this project two players are present, when you click the start button game will start,Dices will move and according to their score which one will be greater it will win the game and it start's again. Here javascript Math.random function and DOM manipulation is Used ,It is a begginer project to make to skill better.![Screenshot (33)](https://user-images.githubusercontent.com/100128956/195383106-b073e8fe-7f99-4dbe-8512-f8e393dcb2c4.png)
```
Comment (aaryan2134) [inline Dicee Challenge - Starting Files/README.md:1]: Please correct few spelling and grammatical errors present

File: Dicee Challenge - Starting Files/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
```
Comment (aaryan2134) [inline Dicee Challenge - Starting Files/README.md:2]: Please add more details and explanation of how you have used various technologies here

**Comment by aaryan2134** [comment]
Are you still working on this? or should I close this?

**Comment by Ritu-2001** [comment]
i am working please give me some time

**Comment by aaryan2134** [comment]
https://github.com/aaryan2134/Engineering-Resources/pull/42
Please correctly format the readme. Currently there are a lot of spaces here and there. Please see the above pull request for reference
You need to add the project in the projects directory

**Pull request closed without merging**